Reading body of a copy of Response doesn't affect the original (#13827)
mdn/content: files/en-us/web/api/fetch_api/using_fetch/index.md

const anotherRequest = new Request(myRequest, myInit);
```

> This is pretty useful, as request and response bodies are one use only. Making a copy like this allows you to make use of the request/response again while varying the `init` options if desired. The copy must be made before the body is read.

Reviewer: Tidy text for "single use"
```suggestion
This is pretty useful, as request and response bodies can only be used once.
Making a copy like this allows you to effectively use the request/response again while varying the `init` options if desired.
The copy must be made before the body is read.
```
